src/app.ts

```diff
@@ -107,9 +112,31 @@ export default class App {
         if (redirect) {
             // const settings = await getSettings();
             // TODO update settings to redirect to the tonomy id website
-            window.location = `https://id.tonomy.com/login?jwt=${token}`;
+            window.location = `https://localhost:3000/login?jwt=${token}`;
```

Comment:
this should come from settings

Reply:
settings isn't global it cant be used here we have a task for this